src/metabase/api/user.clj

```diff
@@ -20,42 +20,35 @@
                      (:is_superuser @api/*current-user*))))
 
 (api/defendpoint GET "/"
-  "Fetch a list of all active `Users` for the admin People page."
+  "Fetch a list of all `Users` for the admin People page."
   []
-  (db/select [User :id :first_name :last_name :email :is_superuser :google_auth :ldap_auth :last_login]
-    :is_active true))
-
-(defn- reactivate-user! [existing-user first-name last-name]
-  (when-not (:is_active existing-user)
-    (db/update! User (u/get-id existing-user)
-      :first_name    first-name
-      :last_name     last-name
-      :is_active     true
-      :is_superuser  false
-      ;; if the user orignally logged in via Google Auth and it's no longer enabled, convert them into a regular user
-      ;; (see Issue #3323)
-      :google_auth   (boolean (and (:google_auth existing-user)
-                                   ;; if google-auth-client-id is set it means Google Auth is enabled
-                                   (session-api/google-auth-client-id)))
-      :ldap_auth     (boolean (and (:ldap_auth existing-user)
-                                   (ldap/ldap-configured?)))))
+  (db/select [User :id :first_name :last_name :email :is_superuser :google_auth :ldap_auth :last_login :is_active]))
```

Comment:
How can we ensure `is_active` is returned from the `POST /api/user` and `PUT /api/user/:id` endpoints as well?